examples/detection/README.md

```diff
@@ -31,3 +31,11 @@ You can generate these visualization results by the following command.
 ```
 $ python visualuze_models.py
 ```
+
+## Notes on writing your own evaluation code
+
+Here are list of important configurations to reproduce results.
+
++ `model.use_preset('evaluate')` changes the score thresholds for NMS.
++ `DetectionVOCEvaluator` should be instantiated with `use_07_metric=True` (default is False), if evaluation is conducted on VOC07 test dataset.
++ When evaluating on VOC dataset, `VOCDetectionDataset` should return information about image difficulty, as the evaluation metric expects that to be included. The dataset returns it by setting `use_difficult=True` and `return_difficult=True`.
```

Comment:
`image difficulty` -> `difficulties of bounding boxes`? (difficulty is not for image)